OpenKh.Tools.ModsManager/Services/ModsService.cs

```diff
@@ -313,8 +313,10 @@ public static bool IsModBlocked(string repositoryName)
                     },
                     OnTransferProgress = (progress) =>
                     {
-                        var nProgress = (progress.IndexedObjects + progress.ReceivedObjects) / (progress.TotalObjects * 2);
+                        var nProgress = ((float)progress.ReceivedObjects / (float)progress.TotalObjects);
                         progressNumber?.Invoke(nProgress);
+
+                        progressOutput?.Invoke("Received Bytes: " + (progress.ReceivedBytes / 1048576) + " MB");
```

Comment:
Should this be a hard-coded number?
IF SO LGTM

Reply:
Yes, bytes to megabytes.